### PR TITLE
docs: Clarify supported MoltenVK version for macOS

### PR DESCRIPTION
**Pull request by DeanBDean**
Resolves https://github.com/EmbarkStudios/rust-gpu/issues/349

### CODE REVIEW

**Comment by eddyb** [comment]
Made this into a draft because it turns out I was wrong about that one bug at least - see the edits to https://github.com/KhronosGroup/SPIRV-Cross/pull/1450#issuecomment-746610707.

File: docs/src/platform-support.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ The `rust-gpu` project currently supports a limited number of platforms and grap
 |-------------------|---------|---------|-------
 | Windows | 10+ | Primary | |
 | Linux | Ubuntu 18.04+ | Primary | |
-| macOS | Catalina (10.15)+ | Secondary | Using [MoltenVK]
+| macOS | Catalina (10.15)+ | Secondary | Using [MoltenVK], requires v1.1.0+
```
Comment (eddyb) [inline docs/src/platform-support.md:14]:
```suggestion
| macOS | Catalina (10.15)+ | Secondary | Using [MoltenVK], requires v1.1.2+
```

as per https://github.com/EmbarkStudios/rust-gpu/issues/349#issuecomment-785804643

**Review by eddyb — CHANGES_REQUESTED** [review]
A new release which contains a fix is now out.

**Review by eddyb — APPROVED** [review]

**Comment by eddyb** [comment]
Waiting for confirmation from anyone else (cc @khyperia as well) about whether we want to do this (including potentially adding more explanation etc.)

**Comment by khyperia** [comment]
LGTM as-is, I don't think we need more explanation (linking to bugs etc.) - like, if someone really cared, they'd look at the git blame, find this PR, get the links to the relevant issues, etc., but the message this says of "things may break if below `v1.1.2`" is both accurate and I think sufficient.

**Review by XAMPPRocky — APPROVED** [review]

**Review by VZout — APPROVED** [review]

**Pull request merged**